### PR TITLE
Run tests in parallel

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: dataplane/dplanerc/interface.go [hunk 1]
```diff
@@ -404,6 +404,17 @@ func (ni *Reconciler) addLAGMember(ctx context.Context, intf ocInterface, member
 		if err := ni.ifaceMgr.LinkSetDown(memberLink); err != nil {
 			log.Warningf("failed to set link %v down: %v", intf.name, err)
 		}
+		for i := 0; i < 3; i++ { // Ensure link is down before continuing.
```
Comment (guoshiuan) [inline dataplane/dplanerc/interface.go:407]: How is 3 selected? can we make it a constant?

File: dataplane/dplanerc/interface.go [hunk 2]
```diff
@@ -404,6 +404,17 @@ func (ni *Reconciler) addLAGMember(ctx context.Context, intf ocInterface, member
...
+			memberLink, err := ni.ifaceMgr.LinkByIndex(memberData.hostifIfIndex)
+			if err != nil {
+				break
```
Comment (guoshiuan) [inline dataplane/dplanerc/interface.go:410]: I suppose the error is due to non-existing link. Should we return error or we just don't care?

File: dataplane/dplanerc/interface.go (hunk 1 above)
Comment (DanG100) [reply dataplane/dplanerc/interface.go]: make a const

File: dataplane/dplanerc/interface.go (hunk 2 above)
Comment (DanG100) [reply dataplane/dplanerc/interface.go]: changed to continue

File: dataplane/dplanerc/interface.go [hunk 3]
```diff
@@ -404,10 +406,10 @@ func (ni *Reconciler) addLAGMember(ctx context.Context, intf ocInterface, member
 		if err := ni.ifaceMgr.LinkSetDown(memberLink); err != nil {
 			log.Warningf("failed to set link %v down: %v", intf.name, err)
 		}
-		for i := 0; i < 3; i++ { // Ensure link is down before continuing.
+		for i := 0; i < ifaceDownRetries; i++ { // Ensure link is down before continuing.
 			memberLink, err := ni.ifaceMgr.LinkByIndex(memberData.hostifIfIndex)
 			if err != nil {
-				break
+				continue
```
Comment (guoshiuan) [inline dataplane/dplanerc/interface.go:412]: add a sleep here too.

**Review by guoshiuan — APPROVED** [review]

**Pull request merged**